### ISSUE TITLE
metric_depth模型怎么使用tensorrt进行推理

### ISSUE THREAD

**Issue by gopin95** [LiheYoung/Depth-Anything#187]
仓库中的metric_depth模型怎么使用tensorrt进行推理？

**Comment by lin-voxvision** [comment]
same question, how can we convert the `depth_anything_metric_depth_outdoor.pt` to an onnx file and then convert it to a trt engine file?

**Comment by pinnintipraneethkumar** [comment]
Hi, @gopin95 @lin-voxvision , you can follow this github page to use depth-anything v2 with tensorrt https://github.com/spacewalk01/depth-anything-tensorrt

**Comment by gopin95** [comment]
Does v2 exist a metric depth estimation model?

**Comment by pinnintipraneethkumar** [comment]
yes, Depth-Anything v2 have metric depth estimation model , https://github.com/DepthAnything/Depth-Anything-V2/tree/main/metric_depth

**Comment by lin-voxvision** [comment]
> Hi, @gopin95 @lin-voxvision , you can follow this github page to use depth-anything v2 with tensorrt https://github.com/spacewalk01/depth-anything-tensorrt

Seems this tensorrt-convert repo only support relative depth model, but I'll try metric anyway. and results will be updated later, Thanks!!
